packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -6,6 +6,7 @@ import 'dart:ui' show Color, hashValues;
 
 import 'package:flutter/foundation.dart';
 import 'package:flutter/services.dart';
+import 'package:flutter/src/material/dialog_theme.dart';
```

Comment:
Should be a local import, like the rest of them:
```
import 'dialog_theme.dart`;
```

Reply:
Done.